src/app/js/admin/parsing/ParsingExcerpt.js

```diff
@@ -53,6 +69,11 @@ export const ParsingExcerptComponent = ({
                         <ParsingExcerptHeaderColumn
                             key={`header_${column}`}
                             column={column}
+                            style={
+                                arrayIsColumnEnrichment[column] && {
+                                    backgroundColor: theme.green.light,
```

Comment:
I think you'd better add an entry in the `styles` const, and reference it in here.